packages/pubsub/src/topic.js

```diff
@@ -456,6 +456,10 @@ Topic.prototype.publish = function(messages, options, callback) {
     method: 'publish',
   };
 
+  if (is.number(options.timeout)) {
+    protoOpts.timeout = options.timeout;
```

Comment:
Oh, one more thing I forgot about, sorry. Can you add a line to the JSDocs above this method that show we accept this option?

```js
 * @param {number} options.timeout - Set a maximum amount of time in
 *     milliseconds before giving up if no response is received.
```